build/.goreleaser.yml

```diff
@@ -8,67 +8,49 @@ builds:
       - linux
     goarch:
       - amd64
+      - arm
```

Comment:
```suggestion
```

Oracle does not provide arm32 client libraries: https://www.oracle.com/database/technologies/instant-client/downloads.html

Additionally, it would be great to check this works e2e (meaning, it is able to scrape some data) before releasing the package.

Reply:
It does provide so I am checking if it works.